services/QuillLMS/app/services/quill_big_query/transformer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module QuillBigQuery
+  class Transformer
+    attr_reader :field_to_lambda, :array_of_hashes
+
+    class UnsupportedBigQueryType < StandardError; end
+    def initialize(fields, array_of_hashes)
+      @field_to_lambda = self.class.build_field_to_lambda_lookup(fields)
+      @array_of_hashes = array_of_hashes
+    end
+
+    LAMBDA_IDENTITY     = ->(val) { val }
+    LAMBDA_TO_BOOLEAN   = ->(val) { val == 'true' }
+    LAMBDA_TO_DATETIME  = ->(val) { DateTime.parse(val) }
+    LAMBDA_TO_F         = ->(val) { val.to_f }
+
+    TYPE_TO_LAMBDA_LOOKUP = {
```

Comment:
You may notice that this list is a superset of https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types 

This is because I noticed empirically, and in non-official BigQuery documentation, that `BOOLEAN` and `FLOAT64` are returned values, in addition to `BOOL` and `FLOAT`.